api/envoy/admin/v3/config_dump.proto

```diff
@@ -61,6 +62,14 @@ message UpdateFailureState {
   // This is the version of the rejected resource.
   // [#not-implemented-hide:]
   string version_info = 4;
+
+  // This resource is not included in the update or being explicitly removed by
```

Comment:
I think I'd phrase this slightly differently:

"This resource has been requested by the client but has either not been delivered by the server or was previously delivered by the server and then subsequently removed from resources provided by the server."

Something like that.

Reply:
Thanks for the suggestion. I have updated the comment and moved it to the `DOES_NOT_EXIST` status.